### PR TITLE
Update Vietnamese Translation P8

### PR DESCRIPTION
**Pull request by Nad-SK**
P8

### CODE REVIEW

File: files/lang/vi.po [hunk 1]
```diff
@@ -5822,359 +5881,504 @@ msgid ""
...
 msgid "A visit and a prayer at the temple raises the morale of your troops."
 msgstr ""
+"Một chuyến viếng thăm và cầu nguyện tại ngôi đền sẽ nâng cao tinh thần cho "
+"quân đội của ngươi."
 
 msgid ""
 "An old Knight appears on the steps of the gazebo. \"I am sorry, my liege, I "
 "have taught you all I can.\""
 msgstr ""
+"Một Hiệp sĩ già xuất hiện trên bậc thềm của vọng lâu. “Tôi xin lỗi, thưa bệ "
```
Comment (ihhub) [inline files/lang/vi.po:6071]: Please always use `\"` for the in-string usage of `"` symbol. Special `“` is not rendered by the engine. The same applies for the string below.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Nad-SK , I left one comment here. Could you please take a look at it?

File: files/lang/vi.po (hunk 1 above)
Comment (Nad-SK) [reply files/lang/vi.po]: Fixed!

**Comment by Nad-SK** [comment]
Hi @ihhub, how to fix this error: not successful when checking.

**Comment by ihhub** [comment]
@Nad-SK , please revert changes for the last commit and fix only the required lines.

**Comment by Nad-SK** [comment]
> @Nad-SK , please revert changes for the last commit and fix only the required lines.

Fixed

**Comment by ihhub** [comment]
@Nad-SK , I can still see 2.3k line changes in yhis pull request. Please revert them.

**Comment by Nad-SK** [comment]
> @Nad-SK , I can still see 2.3k line changes in yhis pull request. Please revert them.

Screenshot (7)
@ihhub, how to fix this??? I can't revert.

**Comment by Nad-SK** [comment]
Hi @ihhub, I revert a 1st Nov commit, but IDK why it has 2k lines changed?
Screenshot (8)

**Comment by Nad-SK** [comment]
Hi, @ihhub thank you for the guideline. It's fixed.

File: files/lang/vi.po [hunk 2]
```diff
@@ -4705,7 +4684,6 @@ msgstr ""
 msgid " button from the "
 msgstr ""
 
-#, fuzzy
 msgid "NEW GAME"
 msgstr ""
 "TẠO\n"
```
[the remark this answers is not in the document]
Comment (Nad-SK) [reply files/lang/vi.po]: Thank @zenseii , It's fixed.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Nad-SK , thank you very much for the update!

**Pull request merged**